docs/changelog.rst

```diff
@@ -206,6 +206,10 @@ Extension support
 
   This makes it easier to support arbitrary encoding in file names.
 
+- The command :command:`mopidy deps` no longer repeats the dependencies of
+  Mopidy itself for every installed extension. This reduces the length of the
+  command's output drastically. (PR: TODO)
```

Comment:
1846

Reply:
Other than that, is this good to merge?

At both my previous and current workplace, we have the convention of commenting+approving if we find minor issues, but don't feel the need to look at the PR again before it is merged, assuming the comments are somewhat addressed. A *request changes* is a quite strong "I'd like to have another look at this once you've addressed these issues".